manifests/tools.pp

```diff
@@ -0,0 +1,18 @@
+# Class for installing MongoDB tools (CLI).
+#
+# == Parameters
+#
+# [ensure] Desired ensure state of the package. Optional.
+#   Defaults to 'true'
+#
+# [package_name] Name of the package to install the tools from. Default
+#   is repository dependent.
+#
+class mongodb::tools (
+  $ensure       = $mongodb::params::package_ensure_tools,
+  $package_name = $mongodb::params::tools_package_name,
+) inherits mongodb::params {
+  anchor { '::mongodb::tools::start': }
```

Comment:
Nowadays we use ```contain``` rather than ```anchor``` but in this case I'd simply move all the content from ```install.pp``` to ```tools.pp```.